src/pages/Loans/components/Dashboard/Dashboard.js

```diff
@@ -13,76 +15,99 @@ import {
-	MINTING_FEE_PERCENT,
-	MIN_LOAN_SIZE,
-	COLLAT_RATIO_PERCENT,
-} = COLLATERAL_PAIR;
+export const Dashboard = ({ walletInfo: { balances, currentWallet }, collateralPair }) => {
+	const [lockedCollateralAmount, setLockedCollateralAmount] = useState(null);
 
-export const Dashboard = ({ walletInfo: { balances } }) => {
 	const { t } = useTranslation();
 
-	const lockedCurrencyBalance = getCurrencyKeyBalance(balances, COLLATERAL_CURRENCY_KEY);
-	const borrowedCurrencyBalance = getCurrencyKeyBalance(balances, BORROWED_CURRENCY_KEY);
-	const lockedCurrencyLoanSize = 1000;
-
-	const numberOfLoans = 0;
-	const borrowedCurrencyLimit = 2500;
-	const borrowedCurrencySupply = 5000;
+	const {
+		collateralCurrencyKey,
+		loanCurrencyKey,
+		interestRatePercent,
+		issueFeeRatePercent,
+		totalOpenLoanCount,
+		minLoanSize,
+		collateralizationRatioPercent,
+		issueLimit,
+		totalIssuedSynths,
+	} = collateralPair;
 
 	const loanInfoItems = [
 		{
 			label: t('loans.dashboard.loan-info.interest-rate'),
-			value: formatPercentage(INTEREST_RATE_PERCENT),
+			value: formatPercentage(interestRatePercent),
 		},
 		{
 			label: t('loans.dashboard.loan-info.minting-fee'),
-			value: formatPercentage(MINTING_FEE_PERCENT),
+			value: formatPercentage(issueFeeRatePercent),
 		},
 		{
-			label: t('loans.dashboard.loan-info.number-of-loans'),
-			value: numberOfLoans,
+			label: t('loans.dashboard.loan-info.number-of-open-loans'),
+			value: totalOpenLoanCount,
 		},
 		{
-			label: t('loans.dashboard.loan-info.currency-limit', { currencyKey: BORROWED_CURRENCY_KEY }),
-			value: formatCurrency(borrowedCurrencySupply),
+			label: (
+				<Trans
+					i18nKey="loans.dashboard.loan-info.currency-limit"
+					values={{ currencyKey: loanCurrencyKey }}
+					components={[<CurrencyKey />]}
+				/>
+			),
+			value: formatCurrency(issueLimit),
 		},
 		{
-			label: t('loans.dashboard.loan-info.currency-supply', { currencyKey: BORROWED_CURRENCY_KEY }),
-			value: formatCurrency(borrowedCurrencyLimit),
+			label: (
+				<Trans
+					i18nKey="loans.dashboard.loan-info.currency-supply"
+					values={{ currencyKey: loanCurrencyKey }}
+					components={[<CurrencyKey />]}
+				/>
+			),
+
+			value: formatCurrency(totalIssuedSynths),
 		},
 		{
 			label: t('loans.dashboard.loan-info.min-loan-size'),
-			value: formatCurrencyWithKey(BORROWED_CURRENCY_KEY, MIN_LOAN_SIZE),
+			value: formatCurrencyWithKey(loanCurrencyKey, minLoanSize),
 		},
 		{
 			label: t('loans.dashboard.loan-info.collateral-size'),
-			value: formatPercentage(COLLAT_RATIO_PERCENT, 0),
+			value: formatPercentage(collateralizationRatioPercent, 0),
 		},
 	];
 
+	useEffect(() => {
+		const getLockedETHBalance = async () => {
+			if (currentWallet) {
+				const balance = await snxJSConnector.provider.getBalance(
+					await snxJSConnector.snxJS.EtherCollateral.contract.address
+				);
+				setLockedCollateralAmount(bigNumberFormatter(balance));
+			}
+		};
+		getLockedETHBalance();
+	}, [currentWallet]);
```

Comment:
do you need current `currentWallet` here? getBalance() will be the same if you change wallet right?

Reply:
Well I want to show this only if the user has a connected wallet. So, if the user changes a wallet - it should rerun this function to reflect the new locked balance.

Reply:
Not a big deal but I wonder what data we should show without the user to have his wallet connected. The number of ETH sent in the contract might be and interesting thing to see?